native_client/javascript/client.ts

```diff
@@ -70,6 +71,13 @@ if (args['scorer']) {
     model.setScorerAlphaBeta(args['lm_alpha'], args['lm_beta']);
   }
 }
+if (args["hot_words"]) {
+	console.error("Adding hot-words %s", args["hot_words"]);
+	for (var word_boost of args["hot_words"].split(",")) {
```

Comment:
Use `let` rather than `var`

Reply:
ok @lissyx  will be done in next few minutes